src/drivers/magnetometer/bmm150/bmm150.cpp

```diff
@@ -415,16 +413,14 @@ BMM150::start()
 	_reports->flush();
 
 	/* schedule a cycle to start things */
-	work_queue(HPWORK, &_work, (worker_t)&BMM150::cycle_trampoline, this, 1);
-
+	ScheduleNow();
```

Comment:
I think ScheduleNow() is the right call here, but just a heads-up, there is a 1ms delay in the current implementation.  I think that the `start()` methods in all of the drivers should just use `ScheduleNow()`... there is a mix of `ScheduleDelay()` and `ScheduleNow()` calls at present.

Reply:
In the old work queue call it's 1 tick, not 1 ms.